modules/installation-gcp-api-services.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+[id="installation-gcp-api-services_{context}"]
+= Enabling API services in GCP
+
+Your Google Cloud Platform (GCP) project requires access to several API services
+to complete {product-title} installation.
+
+.Prerequisites
+
+* You created a project to host your cluster.
+
+.Procedure
+
+* Enable the following required API services in the project that will host your
```

Comment:
s/* Enabling/. Enabling ??

Reply:
You need to enable the services, and single-step procedures use uls instead of ols.